File: coresuite-dotnet-core-console-cs/generator/HelloWorld.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.PageElements;

namespace coresuite_dotnet_core_console_cs.generator
{
    public class HelloWorld
    {
        public static void Run(string outputPdfPath)
        {
            Document document = new Document
            {
                Creator = "HelloWorld",
                Author = "ceTe Software",
                Title = "Hello World"
            };

            // Create a page to add to the document
            Page page = new Page(PageSize.Letter, PageOrientation.Portrait, 54.0f);

            // Uncomment the line below to add a layout grid to the page
            //page.Elements.Add( new LayoutGrid() );

            // Create a Label to add to the page
            string strLabel = "Hello C# World...\nFrom DynamicPDF Generator for .NET\nDynamicPDF.com";
            Label label = new Label(strLabel, 0, 0, 504, 100, Font.Helvetica, 18, TextAlign.Center);

            // Add label to page
            page.Elements.Add(label);

            // Add page to document
            document.Pages.Add(page);

            // Outputs the document to the current web page
            document.Draw(outputPdfPath);
        }
    }
}

File: coresuite-dotnet-framework-console-vb/generator/HelloWorld.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.PageElements

Public Class HelloWorld

    Public Shared Sub Run(outputPdfPath As String)
        'Create a MyDocument and set it's properties
        Dim MyDocument As Document = New Document With {
            .Creator = "HelloWorld.aspx",
            .Author = "ceTe Software",
            .Title = "Hello World"
        }

        'Create a MyPage to add to the MyDocument
        Dim MyPage As Page = New Page(PageSize.Letter, PageOrientation.Portrait, 54.0F)

        'Uncomment the line below to add a layout grid to the MyPage
        'MyPage.Elements.Add(New LayoutGrid())

        'Create a label to add to the MyPage
        Dim Text As String = "Hello ASPX VB.NET World..." & vbCrLf & "From DynamicPDF Generator for .NET" & vbCrLf & "DynamicPDF.com"
        Dim MyLabel As Label = New Label(Text, 0, 0, 504, 100, Font.Helvetica, 18, TextAlign.Center)

        'Add label to MyPage
        MyPage.Elements.Add(MyLabel)

        'Add MyPage to MyDocument
        MyDocument.Pages.Add(MyPage)

        'Outputs the MyDocument to the current web MyPage
        MyDocument.Draw(outputPdfPath)
    End Sub
End Class

